defaults/main.yml

```diff
@@ -11,6 +12,7 @@ django_stack_gcorn_port: 8000
 django_stack_gcorn_log: /var/log/{{django_stack_app_name}}/app
 django_stack_gcorn_app: freedompress.wsgi
 django_stack_gcorn_app_settings: freedompress.settings.production
```

Comment:
Sure, as long as the "global" var is also namedspace, e.g. `{{ django_stack_global_app_name }}`.